module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,15 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::vector<std::string> removeVowels(std::vector<std::string>& vec) {
+    std::vector<char> vowelVec{'a', 'e', 'i', 'o', 'u', 'y'};
+
+    for (std::string& str : vec) {
+        for (const char& vowel : vowelVec) {
+            str.erase(std::remove(str.begin(), str.end(), vowel), str.end());
+            str.erase(std::remove(str.begin(), str.end(), vowel - 32), str.end());
+        }
+    }
+    return vec;
+}
```

Comment:
missing newline

Reply:
I don't understand why we need a newline. It's last line of header.

Reply:
Some non-typical compilers may have a problem with it. It's a good practice to always have a new line at the end of the file.